src/adapter/src/catalog/objects.rs

```diff
@@ -742,20 +762,20 @@ impl CatalogItem {
         }
     }
 
-    pub fn desc(&self, name: &FullItemName) -> Result<Cow<RelationDesc>, SqlCatalogError> {
+    pub fn desc(&self, name: String) -> Result<Cow<RelationDesc>, SqlCatalogError> {
```

Comment:
This name is only used for displaying the error. The item does store a `name`, but it's of type `QualifiedItemName` which should not be displayed. OOC would it be bad to store something like a `display_name` as well in the `CatalogItem`? Then this `desc` wouldn't need this to be passed.

Reply:
Agreed it's weird, but I think there's some logic behind storing the `QualifiedItemName` rather than the `FullItemName`. Can of worms to open, at least. I pushed up a local fix that splits out a non-erroring `desc_opt` method, which cleans up the name invention thing, which should be good enough for now.

Reply:
👍